pkg/webui/console/containers/fetch-table/index.js

```diff
@@ -89,6 +90,58 @@ class FetchTable extends Component {
     this.debounceCancel = cancel
   }
 
+  static propTypes = {
+    addMessage: PropTypes.message,
+    fetching: PropTypes.bool,
+    fetchingSearch: PropTypes.bool,
+    handlesPagination: PropTypes.bool,
+    headers: PropTypes.arrayOf(
+      PropTypes.shape({
+        displayName: PropTypes.message.isRequired,
+        getValue: PropTypes.func,
+        name: PropTypes.string,
+        render: PropTypes.func,
+        centered: PropTypes.bool,
+        sortable: PropTypes.bool,
+        width: PropTypes.number,
+      }),
+    ),
+    itemPathPrefix: PropTypes.string,
+    items: PropTypes.arrayOf(PropTypes.shape({})),
+    mayAdd: PropTypes.bool,
+    pageSize: PropTypes.number,
+    pathname: PropTypes.string,
```

Comment:
```suggestion
    pathname: PropTypes.string.isRequired,
```
Should work, I believe.